### PR TITLE
Feature/624 chart most measurements monitoring report

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-624](https://jira.epa.gov/browse/HMW-624)

## Main Changes:
* When entering a Monitoring Report page, the characteristic with the most measurements is selected from the table and charted.
* The characteristics table is sorted by selected characteristics by default.

## Steps To Test:
1. Go to http://localhost:3000/monitoring-report/STORET/MDE_TMDL/MDE_TMDL-AR19/.
2. Ensure that a characteristic with 4 detectable results is selected (there are multiple characteristics with 4 detectable results, and only the first found is selected; notably, "Decachlorobiphenyl", which has 6 total results but 0 detectable results, should **not** be selected).

### CODE REVIEW

File: app/client/src/components/pages/MonitoringReport.js [hunk 1]
```diff
@@ -1485,7 +1502,7 @@ function CharacteristicsTableSection({
             autoResetFilters={false}
             autoResetSortBy={false}
             data={tableData}
-            defaultSort="name"
+            defaultSort="selected"
```
Comment (cschwinderg) [inline app/client/src/components/pages/MonitoringReport.js:1505]:
```suggestion
            defaultSort="measurementCount"
```

I know the ticket says to sort by selected, but I think we really need to sort by measurment count per Kiki's notes on 9/14.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good. Just one  small change related to the sorting.

File: app/client/src/components/pages/MonitoringReport.js (hunk 1 above)
Comment (maxdiebold-erg) [reply app/client/src/components/pages/MonitoringReport.js:1505]: Good call, I missed that. One reservation I had with sorting by measurement count is that the selected characteristic could still get lost if there are many with the same count. However, it looks like there is multi-sorting, so I can look into that.

**Comment by maxdiebold-erg** [comment]
Since the default sort is only used in one place, it made more sense to pass complete control of the default sort array.

I also added in a small note indicating how to sort the table.

**Review by cschwinderg — APPROVED** [review]
Works well! 

I went ahead and deleted the `ReactTable1.js` file, since it was empty.

**Pull request merged**